exammode/viewbase.py

```diff
@@ -0,0 +1,3 @@
+from lib.viewbase import BaseMixin, BaseTemplateView
+
+from .models import ExamSession, ExamAttempt
```

Comment:
Is this file needed if it only imports other modules?

Reply:
This file is not imported anywhere and has no functions, so it is currently useless and removing it should not have any side effects.

Reply:
This file is removed.